### PR TITLE
Update dygraph lib info, auto-update w/ v2.0.0

### PR DESCRIPTION
**Pull request by maruilian11**
Pull request for issue: #10358 
Related issue(s): #10360 
@cdnjs/intern2 Please help me review this PR. Thanks!

### CODE REVIEW

**Comment by maruilian11** [comment]
@pvnr0082t This issue was opened by the author, and he has tried to send the commits in #10360, but he hoped I can send the PR for him. I think it's better to let him know the latest commit state.

**Comment by maruilian11** [comment]
@cdnjs/intern2 I have update the commit msg, please help me review this PR, thanks!

File: ajax/libs/dygraph/package.json [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-    "charts",
-    "interactive"
+    "timeseries",
+    "chart",
+    "canvas",
+    "visualization",
+    "dygraphs"
+  ],
+  "npmname": "dygraphs",
+  "npmfilemap": [
```
Comment (PeterDaveHello) [inline ajax/libs/dygraph/package.json]: @maruilian11 @pvnr0082t big mistake, both of you didn't test it?

Comment (PeterDaveHello) [inline ajax/libs/dygraph/package.json]: Since this is a high priority issue, so I just merged and fixed locally, please be careful next time.

**Comment by maruilian11** [comment]
Sorry for that, I copied the config I tested before, so I didn't test it again. But the copy process turn the capital to lower case, and I didn't notice that. I will be more careful next time.

**Pull request merged**